src/gpio/stm32l496ag.rs

```diff
@@ -0,0 +1,65 @@
+//! GPIO specific to STM32L496AG
```

Comment:
Can you please add notice that particular feature is required to use this module e.g.

This module is enabled by feature `<NAME>`